server/prisma/schema.prisma

```diff
@@ -36,10 +36,30 @@ model SessionUserGifts{
 model Users {
   id                       Int                        @id @default(autoincrement())
   email                    String                     @unique
-  firstName                String
-  lastName                 String
+  firstName                String?
+  lastName                 String?
   password                 String
   sessionUserGifts         SessionUserGifts[]
+  tokens                   Token[]
+}
+
+model Token {
+ id         Int       @id @default(autoincrement())
+ createdAt  DateTime  @default(now())
+ updatedAt  DateTime  @updatedAt
+ type       TokenType
```

Comment:
Can we switch the email token logic for a user being able to sign in with a username and password?

Reply:
@mdwiltfong thanks so much for all your help today! I updated the schema to use just one `jwtToken` and the user can login with the user and password.